sugar-iconify.md

```diff
@@ -0,0 +1,143 @@
+#Sugar Iconify
+
+##Description
+
+This script will create Sugar-compatible SVG icons from input.svg by
```

Comment:
You say "This script" but the reader doesn't yet know what script you are referring to.